shell-completion/bash/kernel-install

```diff
@@ -42,6 +44,18 @@ _kernel_install() {
             ;;
     esac
 
+    if [[ "${COMP_WORDS[1]}" = "remove" ]] && [[ $cur = -* ]]; then
+        COMPREPLY=( $(compgen -W '--entry-type' -- "$cur") )
+        return 0
+    fi
+
+    case "$prev" in
+        --entry-type)
+            COMPREPLY=( $(compgen -W "$entry_types" -- "$cur") )
```

Comment:
Hmm, if I type `kernel-install remove --entry-type <TAB>`, I don't get any completions. As soon as I add `t`, then the completion works. I have no idea what that is or how to fix it. If you don't know either, we can leave it as is, it's not a big deal.